vsdownload.py

```diff
@@ -538,7 +541,10 @@ def moveVCSDK(unpack, dest):
     # files to be removed.
     makedirs(os.path.join(dest, "kits"))
     mergeTrees(os.path.join(unpack, "VC"), os.path.join(dest, "VC"))
-    mergeTrees(os.path.join(unpack, "Program Files", "Windows Kits", "10"), os.path.join(dest, "kits", "10"))
+    if sys.platform == "win32":
+        mergeTrees(os.path.join(unpack, "Windows Kits", "10"), os.path.join(dest, "kits", "10"))
+    else:
+        mergeTrees(os.path.join(unpack, "Program Files", "Windows Kits", "10"), os.path.join(dest, "kits", "10"))
```

Comment:
So there's a difference between the Windows native `msiexec` and the third party `msiextract`, where the latter includes an extra `Program Files/` toplevel directory here, while the former doesn't? That's tricky...

Can we reduce the amount of duplicated code here a bit, by doing e.g. this?
```
kitsPath = unpack
if sys.platform == "win32:
    kitsPath = os.path.join(kitsPath, "Program Files")
kitsPath = os.path.join(kitsPath, "Windows Kits", "10")
mergeTrees(kitsPath, os.path.join(dest, "kits", "10"))
```